libzhlgen/test/functions/mac/1.6.13/ScoreKeeper.zhl

```diff
@@ -0,0 +1,62 @@
+"554889e54157415641554154534883ec284889fb48c70300":
+cleanup __amd64 void ScoreKeeper::constructor(ScoreKeeper *this);
+"554889e54157415641554154534883ec384889f1":
+cleanup __amd64 std::pair<int, int> ScoreKeeper::GetShipId(ScoreKeeper *this, const std::string& blueprintName);
+"554889e553504889fb4863c2488d0440":
+cleanup __amd64 std::string ScoreKeeper::GetShipBlueprint(ScoreKeeper *this, int index);
+"554889e541565389f34989fe488d057d":
+cleanup __amd64 void ScoreKeeper::SaveGame(ScoreKeeper *this, int fd);
+"554889e5415741564154534881ec8001":
+cleanup __amd64 void ScoreKeeper::LoadGame(ScoreKeeper *this, int fd);
+"554889e54157415641554154534883ec684189d44189f648":
+forceDetour cleanup __amd64 bool ScoreKeeper::GetShipUnlocked(ScoreKeeper *this, int shipId, int shipVariant);
```

Comment:
`forceDetour` might not be needed, see other comment on it. Also must validate there is sufficient room for forcing the detour in this binary.